django_mongodb/features.py

```diff
@@ -86,6 +86,9 @@ class DatabaseFeatures(BaseDatabaseFeatures):
         "auth_tests.test_views.LoginTest.test_login_session_without_hash_session_key",
         # GenericRelation.value_to_string() assumes integer pk.
         "contenttypes_tests.test_fields.GenericRelationTests.test_value_to_string",
+        # INTPYTHON-348 add support for QuerySet.raw_mql()
+        # https://github.com/mongodb-labs/django-mongodb/pull/183
+        "raw_query.tests.RawQueryTests",
```

Comment:
The `@expectedFailure` decorator can't be applied on classes, only test methods. But if you put the tests in expected failures, you won't be able to see how they fail so you can fix them...?